distribution/pom.xml

```diff
@@ -11,22 +11,6 @@
    <name>Infinispan Distribution</name>
    <packaging>pom</packaging>
 
-   <properties>
-      <analytics.code>
----&gt;
--&lt;!--
--&lt;script type='text/javascript'&gt;
--var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
--document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
```

Comment:
The analytics code has been refactored [in this PR](https://github.com/infinispan/infinispan/pull/4440). I would prefer if you didn't touch it and integrate my PR instead!